src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write your code here
+const content = document.querySelectorAll('.population');
+let total = 0;
+let avarage = 0;
+const arr = [];
+
+for (const child of content) {
+  const re = /,/gi;
+
+  arr.push(+child.textContent.replace(re, ''));
+
+  total = arr.reduce((a, b) => {
+    return a + b;
+  });
```

Comment:
You can move this outside of for loop. Use loop to fill the values array, and then when loop is done calculate total one time, instead of recalculating it each iteration